core/ecschema-metadata/src/Metadata/KindOfQuantity.ts

```diff
@@ -141,7 +141,7 @@ export class KindOfQuantity extends SchemaItem {
     return new OverrideFormat(parent, precision, unitLabelOverrides);
   }
 
-  private async processPresentationUnits(presentationUnitsJson: string | string[]): Promise<void> {
+  protected async processPresentationUnits(presentationUnitsJson: string | string[]): Promise<void> {
```

Comment:
I'm not sure about that one. We already have a public method addPresentationFormat and I think we should rather use that one instead of extending our public api. We might need to discuss that further.